spec.md

```diff
@@ -433,26 +432,25 @@ that performs actions across multiple types of events. To support such use
 cases, the event producer will need to add additional identity attributes to the
 "context attributes" which the event consumers can use to correlate this event
 with the other events. If such identity attributes happen to be part of the
-event "data", the event producer SHOULD also add the identity attributes to the
+data payload, the event producer SHOULD also add the identity attributes to the
 "context attributes" so that event consumers can easily access this information
-without needing to decode and examine the event data. Such identity attributes
+without needing to decode and examine the data payload. Such identity attributes
 can also be used to help intermediate gateways determine how to route the
 events.
 
-## Data Attribute
+## Data Payload
 
 As defined by the term [Data](#data), CloudEvents MAY include domain-specific
-information about the occurrence. When present, this information will be
-encapsulated within the `data` attribute.
+information about the occurrence. When present, this information SHALL be
+encapsulated by the data payload.
 
-### data
+The data payload is optional, some events MAY have no corresponding
+domain-specific information, rather, all information relevant to the Occurance
+SHOULD be found in the Context, such as the `type` attribute.
```

Comment:
In general, I don't understand why it is stated three times that the payload is optional:

"CloudEvents MAY include"
"payload is OPTIONAL"
"events MAY have no"

> all information relevant to the Occurance SHOULD be found in the Context, such as the `type` attribute.

I find that this section nudges the reader towards leaving data empty, and encourages to cram more into the context. I think it is at odds with what we say in the primer.

---

TL;DR: What value does this paragraph bring? What do we miss if we remove lines 447 to 449?